### PR TITLE
API base url

### PR DESCRIPTION
**Pull request by deepak-shah-np**
API base url as runtime config for the app.

### CODE REVIEW

File: src/composables/useFetchData.ts [hunk 1]
```diff
@@ -9,8 +9,11 @@ export function useFetchData() {
     error.value = null;
 
     try {
+      const runtime = useRuntimeConfig();
+      const baseUrl = runtime.public.apiBaseUrl;
       const queryString = new URLSearchParams(params as any).toString();
-      url="https://guy-mangroves.servirglobal.net/map/get-extent-layer/";
+      url=`${baseUrl}/map/get-extent-layer/`;
+      console.log("url",url)
```
Comment (billyz313) [inline src/composables/useFetchData.ts:16]: console.log should be removed for production

**Review by billyz313 — CHANGES_REQUESTED** [review]
Line 16 in useFetchData.ts "console.log...." should be removed for production.

File: src/composables/useFetchData.ts (hunk 1 above)
Comment (deepak-shah-np) [reply src/composables/useFetchData.ts]: Removed!
Thank you for pointing!

**Review by billyz313 — APPROVED** [review]
Thank you

**Pull request merged**